components/org.wso2.carbon.identity.scim2.provider/src/main/java/org/wso2/carbon/identity/scim2/provider/util/SCIMProviderConstants.java

```diff
@@ -40,5 +40,11 @@
     public static final String HTTP_VERB = "HTTP_VERB";
     public static final String SEARCH = ".search";
     public static final String DEFAULT_USERNAME = "admin";
+    public static final String ADD = "add";
+    public static final String REMOVE = "remove";
+
+    public static final int HTTP_STATUS_SUCCESS = 200;
```

Comment:
No need to define again. Use the constants defined in ResponseCodeConstants.java in charon